shared/chat/conversation/info-panel/menu/index.js

```diff
@@ -38,11 +39,25 @@ const InfoPanelMenu = (props: Props) => {
       onClick: props.onInvite,
     },
   ]
-  const channelItem = {
-    title: props.isSmallTeam ? 'Make chat channels...' : 'Manage chat channels',
-    onClick: props.onManageChannels,
-    subTitle: props.isSmallTeam ? 'Turns this into a big team' : undefined,
-  }
+  const channelItem = props.isSmallTeam
+    ? {
+        onClick: props.onManageChannels,
+        subTitle: 'Turns this into a big team',
+        title: 'Make chat channels...',
+      }
+    : {
+        onClick: props.onManageChannels,
+        title: 'Subscribe to channels...',
+        view: (
+          <Box style={globalStyles.flexBoxRow}>
+            {props.badgeSubscribe && <Box style={styleBadge} />}
+            <Text style={isMobile ? styleText : {}} type={isMobile ? 'BodyBig' : 'Body'}>
```

Comment:
likely better to have styleText's definition have the isMobile check instead of here